playbooks/post_deploy.yml

```
@@ -48,108 +38,14 @@
 
 # Call site-specific playbook
 - import_playbook: "{{ cc_ansible_site_dir }}/post_deploy.yml"
-
 # Initialize OpenStack entities. This is done after site-specific playbooks,
 # because those playbooks may affect networking, which may not be available
 # until here.
+
 - hosts: control
+  roles:
+    - post_ironic
```

Maybe add [a condition](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#conditionals-with-roles)?
```suggestion
    - role: post_ironic
      when: enable_ironic | bool
```